.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] kindly add `node_modules/` in the .ignore file as it's in the requirement of the project